src/main/java/alibek/User.java

```diff
@@ -0,0 +1,33 @@
+package alibek;
+
+import java.util.*;
+
+
+public class User {
+    String name;
+    User(String name) throws Exception {
+        validateName(name);
+        this.setName(name);
+    }
+    User() {
+
+    }
+    public String getName() {
+        return this.name;
+    }
+    public void setName(String name) {
+        this.name = name;
+    }
+    public boolean isMe(String name) {
```

Comment:
I don't think you need this method here. If you provide a getName method, then the consumer of your User class can just use getName to check that condition.